fog/sql_recovery_db/src/lib.rs

```diff
@@ -1412,10 +1509,10 @@ mod tests {
         db.decommission_ingest_invocation(&invoc_ids[1]).unwrap();
 
         // Add two missing block records.
-        db.report_missed_block_range(&BlockRange::new(10, 20))
-            .unwrap();
-        db.report_missed_block_range(&BlockRange::new(30, 40))
-            .unwrap();
+        // db.report_missed_block_range(&BlockRange::new(10, 20))
```

Comment:
Leftovers. Unless replaced by a different test, we should include reporting of a lost key and see that it translates to a missed blocks event.

Reply:
thank you, done